pkg/downloader/manager.go

```diff
@@ -537,12 +544,20 @@ func tarFromLocalDir(chartpath string, name string, repo string, version string)
 		return "", fmt.Errorf("wrong format: chart %s repository %s", name, repo)
 	}
 
-	origPath, err := filepath.Abs(strings.TrimPrefix(repo, "file://"))
-	if err != nil {
-		return "", err
+	p := strings.TrimPrefix(repo, "file://")
```

Comment:
Thank you for pointing it out. I was thinking of the same. Now it becomes a more clear logic unit so a function will increase readability.